### DISCUSSION TITLE
Bot posts SetPlayerPos command directly to Discord

### DISCUSSION THREAD

**Discussion by Dalo1101** [gameserverapp/Platform#733] [Suggestions]
When a report is opened and the bot sends this to the Discord channel... it would be nice if the SetPlayerPos command was included directly in this post, so that you don't have to go to the GSA website first

**Comment by mvaessen** [comment]
<img width="388" height="234" alt="Screenshot 2026-02-26 at 14 28 44" src="https://github.com/user-attachments/assets/0c9e8555-d749-4e84-865f-5f78ca073bad" />

This will be added later today
